Doc/library/re.rst

```diff
@@ -931,8 +931,8 @@ form.
    This is useful if you want to match an arbitrary literal string that may
    have regular expression metacharacters in it.  For example::
 
-      >>> print(re.escape('python.exe'))
-      python\.exe
+      >>> print(re.escape('http://www.python.org'))
```

Comment:
With this change, the example illustrates some of the characters that are no longer being escaped in a somewhat important use case. I think that, with it, the total change can be minimal and the other examples could be kept as they were. Your input is appreciated, @serhiy-storchaka.

Reply:
I like this. The example demonstrates the use for an arbitrary literal string. It contains letters, which should not be escaped, special characters which must be escaped, and characters which was escaped in old Python versions, but which are not escaped now.

Please restore the third example.